tarodb/assets_store.go

```diff
@@ -1265,6 +1266,10 @@ func (a *AssetStore) LogPendingParcel(ctx context.Context,
 		txnID, err := q.UpsertChainTx(ctx, ChainTx{
 			Txid:  newAnchorTXID[:],
 			RawTx: anchorTxBytes,
+			HeightHint: sql.NullInt32{
+				Int32: int32(anchorTxHeightHint),
+				Valid: true,
```

Comment:
Independent of the comment above, you can use `sqlInt32` here (in `squitils.go`).